department-of-veterans-affairs/va.gov-cms
# VACMS-3385: Add content types to alert ref field.

## Description
See #3385 

## Testing done
Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/98240741-e0327e80-1f37-11eb-98bf-836bcb6d7978.png)

## QA steps

- As an administrator, visit `/admin/structure/block/block-content/manage/alert/fields/block_content.alert.field_node_reference` and visually verify that the following types have been added to checked options:
--  Checklist
--  Step-by-step
--  Media list - Video
--  Media list - Images
--  Q&A
--  FAQ - multiple Q&As
--  Learning Center Detail Page
--  Landing Page
